SConstruct

```diff
@@ -65,7 +65,16 @@ num_process = GetOption('num_jobs')
 pipeline = env.Command(os.path.join(REPO_ROOT, "shared", "ci_hsc_output"), raws,
                        ["bin/pipeline.sh {}".format(num_process)])
 
-everything = [butler, instrument, skymap, external, raws, pipeline]
+tests = []
+for root, _, files in os.walk(os.path.join(PKG_ROOT, "tests")):
```

Comment:
I'm not sure we really want to walk subdirectories recursively here.  Maybe just `os.listdir` or even `env.Glob` instead?